recipes/xgrads/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "xgrads" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d354bd5c69fbd05bcbbd34ac15d589a6592305dfd1769addd4a0cfeba04d4d97
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - xarray
+    - dask
+    - numpy
+    - pyproj
+    - numba
```

Comment:
This one is not listed upstream, is it really needed?

Reply:
Hi, @BastianZim It is necessary to run the tests and Jupiter notebooks. But not so for the package itself.